bson/tests/test_int.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+from unittest import TestCase
+
+from bson import dumps, loads
+
+
+class TestInt(TestCase):
+    def setUp(self):
+        self.goodRquestDict = {
```

Comment:
Could you change to use underscore `good_request_dict`?
I will merge after this! :)

Reply:
Done.
@Parkayun